Remove request parameter in Typed Replicator messages, #27115 (#27117)
akka/akka: akka-docs/src/main/paradox/typed/distributed-data.md

:  @@snip [ReplicatorTest.java](/akka-cluster-typed/src/test/java/akka/cluster/ddata/typed/javadsl/ReplicatorTest.java) { #sample }

> When we start up the actor we subscribe it to changes for our key, this means that whenever the replicator sees a change

Reviewer: ```suggestion
When we start up the actor we subscribe it to changes for our key, meaning whenever the replicator observes a change
```

> for the counter our actor will get a @scala[`Replicator.Changed[GCounter]`]@java[`Replicator.Changed<GCounter>`]. Since

Reviewer: ```suggestion
for the counter our actor will receive a @scala[`Replicator.Changed[GCounter]`]@java[`Replicator.Changed<GCounter>`]. Since
```
